### PR TITLE
Inginious 0.40

### PR DESCRIPTION
**Pull request by zimbatm**
###### Motivation for this change

unbreak the package so that the module can be evaluated. See the comments on 611116823a13930caf2bf0402300b9b523efb92b

/cc @layus @grahamc 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -14851,12 +14851,13 @@ in {
   psd-tools = callPackage ../development/python-modules/psd-tools { };
 
   psutil = buildPythonPackage rec {
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: this bump will require a separate PR

**Review by FRidh — CHANGES_REQUESTED** [review]

**Comment by zimbatm** [comment]
nox-review is failing with:
```
warning: no previously-included files matching '*.py[co]' found anywhere in distribution
no previously-included directories found matching 'docs/_build'
builder for ‘/nix/store/q5pqx45v5l4c8v752l5c44pl14a0njxf-python2.7-oslo.reports-0.6.0.drv’ failed with exit code 1
```

**Comment by layus** [comment]
Could we remove this package and the corresponding module ? It is a tool under development with a tendency to use cutting edge versions of all the python dependencies. It is quite difficult to maintain, and I have little time to do so ATM.

**Comment by layus** [comment]
Plus, @grahamc fixed the module eval in #31973

**Comment by FRidh** [comment]
psutil is in staging

**Pull request closed without merging**